src/python/WMCore/Services/WMArchive/DataMap.py

```diff
@@ -207,6 +211,10 @@ def typeCastPerformance(performDict):
     for key in PERFORMANCE_TYPE:
         if key in performDict:
             for param in PERFORMANCE_TYPE[key]:
+                if key == 'cmssw':
+                    # so far we skip validation of cmssw metrics values
+                    newPerfDict[key] = performDict[key]
```

Comment:
If these are new metrics to be provided to the monitoring system, why do we skip this validation?

One of my fears is that, once we start validating it, that we get different metric data types for different CMSSW releases. If that happens, this will crash until someone looks and resolve the issue. Perhaps the way to deal with this would be to catch an exception and simply provide the value as is, thus avoiding human action.

Reply:
This is why I skipped validation, and in fact I do not know how to properly do it.